lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -270,6 +270,13 @@
             - Name of the storage domain this virtual machine lease reside on.
             - NOTE - Supported since oVirt 4.1.
         version_added: "2.4"
+    custom_compatibility_version:
+        description:
+            - " Enables a virtual machine to be customized to its own compatibility version.  If
+            `custom_compatibility_version` is set, it overrides the cluster's compatibility version
```

Comment:
Here please use C(custom_compatibility_version) instead of `custom_compatibility_version`.